src/com/tony/server/Main.java

```diff
@@ -1,8 +1,6 @@
 package com.tony.server;
 
-import com.tony.server.response.GetResponse;
-import com.tony.server.response.OptionsResponse;
-import com.tony.server.response.PutPostResponse;
+import com.tony.server.response.*;
```

Comment:
Don't use wildcard imports. Be explicit about what you're importing even if that means three lines of imports vs one.